walkbox.go

```diff
@@ -57,14 +57,14 @@ func (w *WalkBox) isConvex() bool {
 	return totalCrossProduct != 0
 }
 
-// Enable sets the enabled state of the WalkBox.
```

Comment:
removed from WalkBox API because, enable/disable a walkbox has other effects as calculate the itinerary matrix again.
This function is provided by WalkBoxMatrix